### DISCUSSION TITLE
Generated class images are all brown and tan squares.

### DISCUSSION THREAD

**Discussion by FugueSegue** [d8ahazard/sd_dreambooth_extension#500] [Q&A] [answered]
UPDATE (13 Dec. 2022):

I believe I have figured out what I was doing wrong. I think I was using the incorrect YAML files for the various checkpoint models.

You can download YAML files that work with v2-1 checkpoints from this webpage:

https://github.com/Stability-AI/stablediffusion/tree/main/configs/stable-diffusion

Use the v2-inference.yaml with the v2-1_512-ema-pruned.ckpt and v2-1_512-nonema-pruned.ckpt files.

Use the v2-inference-v.yaml with the v2-1_768-ema-pruned.ckpt and v2-1_768-nonema-pruned.ckpt files.

In order to properly use a YAML file, copy the appropriate YAML file to the same directory as the appropriate CKPT file. Then you rename the YAML so that it has the exact same name as corresponding CKPT file. Do not change the extensions of the file names.

I hope this helps anyone who had the same trouble that I had.

What I do not know is how to properly use the other YAML files from Stability AI webpage I linked above:

v2-inpainting-inference.yaml
v2-midas-inference.yaml
x4-upscaling.yaml

If anyone knows how to use the three YAML files above, please let me know and I will update this post.

____________________

ORIGINAL POST:

I am using the v2-1_512-nonema-pruned.ckpt for the training. I also have placed a copy of the v2-inference.yaml file in the same directory and renamed it v2-1_512-nonema-pruned.yaml. I have read that this is the correct arrangement. So I don't think that's the problem.

I suspect my command line is the culprit:

COMMANDLINE_ARGS="--listen --xformers"

Suggestions?

ADDITIONAL:
I manually generated class images using txt2img with the same settings used in the Concepts tab. I was able to train a model with v2-1_512-nonema-pruned.ckpt. But the resulting model produces nothing but these brown images.

**Comment by FugueSegue** [comment, marked as answer]
I believe I have learned the answer to my question. See the original post.

**Comment by zahdab** [comment]
hello, i am currently facing this issue i did put the proper yaml file in the 512 directory but still getting the brown images

**Comment by zahdab** [comment]
i tried flipping the yaml files between the 768 and 512 models to see if i had the misplaced but it wasnt the case ... i got noisy pixels ... so i am pretty sure i have the right yaml file in the right folder ... though all i am getting for classification images is brown images for 512 and black images fo 768

**Reply by mcschippenhorst** [reply]
same here

**Comment by zahdab** [comment]
@GunnarQuist  did you try to generate the class reg images again via the dreambooth training, or did you rely on the ones you generated manually? maybe that's how your model got working?
